v1/github/org/old-secrets.rego

```diff
@@ -33,12 +33,12 @@ allow {
 
 reason = v {
 	allow
-	v := "No secrets are older than max_secret_age months"
+	v := "No secrets are older than the threshold"
 }
```

Comment:
How does the person getting the log know the threshlold?

Reply:
He has it as an input + in valint verbose logs, but we can consider mentioning it in the results as well.